tst/EnergyPlus/unit/SetPointManager.unit.cc

```diff
@@ -1517,7 +1590,8 @@ TEST_F(EnergyPlusFixture, SingZoneCoolHeatSetPtMgrZoneInletNodeTest)
         "   **  Fatal  ** InitSetPointManagers: Errors found in getting SetPointManager input.",
         "   ...Summary of Errors that led to program termination:",
         "   ..... Reference severe error count=2",
-        "   ..... Last severe error=SetpointManager:SingleZone:Cooling=\"COOLING SUPPLY AIR TEMP MANAGER 1\", The zone inlet node of ZNF1 INLET NODE",
+        "   ..... Last severe error=SetpointManager:SingleZone:Cooling=\"COOLING SUPPLY AIR TEMP MANAGER 1\", The zone inlet node of ZNF1 INLET "
+        "NODE",
```

Comment:
FWIW, some of these error snippets that have to match _exactly_ could be modified to use [`compare_err_stream_substring`](https://github.com/NREL/EnergyPlus/blob/92ba541ca525940013c228b84b7644d31212118f/tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.hh#L193).  That way you can just make sure it has a reasonable message, and not have these full text blocks that have to match exactly.

I know you didn't do a functional change here, it just line wrapped, but I'm saying it anyway :)